src/20_Components/amp-app-banner.html

```diff
@@ -0,0 +1,91 @@
+  <!--
+    The `amp-app-banner` component uses the `prefer_related_applications` key from the web app manifest. So, you could use exactly the same manifest for your actual site and get the native Chrome banner too.
+  -->
+  <link rel="manifest" href="/amp-app-banner-manifest.json">
+  <!--
+    Deep links into your apps are calculated based on you AMP's canonical link. Here we link to an article on Medium as we don't have a native AMP by Example app. 
+  -->
+  <link rel="canonical" href="https://medium.com/google-developers/how-to-avoid-common-mistakes-when-publishing-accelerated-mobile-pages-9ea61abf530f" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+<style amp-custom>
+#banner {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+#banner > #banner-action, #banner > #banner-logo {
+  flex: 0;
+}
+
+#banner > #banner-text {
+  flex: 1;
+  font-size: 14px;
+  padding: 0 8px;
+}
+
+#banner button[open-button] {
+  color: #00ab6b;
+  width: 120px;
+  height: 49px;
+  padding: 0;
+  margin: 0;
+}
+</style>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+The `amp-app-banner` can be fully customized by the developer as long as the height doesn't exceed 100px.  One required UI element is a button with the attribute `[open-button]`, which triggers the app install, or opens the deep-link if the app is already installed.
+
+  -->
+  <amp-app-banner layout="nodisplay" id="banner">
+    <div id="banner-logo">
+      <amp-img src="https://cdn-images-1.medium.com/max/800/1*JLegdtjFMNgqHgnxdd04fg.png" width="53" height="49" layout="fixed"></amp-img>
+    </div>
+    <div id="banner-text">Learn more about AMP in the Medium App.</div>
+    <div id="banner-action">
+      <button open-button>View in app</button>
+    </div>
+  </amp-app-banner>
+
+  <!-- #### Testing --> 
+  <!--
+    The amp-app-banner will only show in browsers that don't feature a native app install banner. This means Chrome on Android or Safari on iOS will not show the amp-app-banner, but the native install banner. For testing amp-app-banner you can open this page:
```

Comment:
I find the use of "native" is a bit confusing here, especially since it occurs adjacent to "app." (Isn't the point to install a "native app"?) Maybe something like "… in browsers that don't feature a browser-generated app install banner" or "… in browsers that don't provide their own app install banner" would be clearer.